content/faq/app/crash/index.uk.md

```diff
@@ -4,7 +4,7 @@ description: ""
 updated: "2024-06-12"
 
 taxonomies:
-  faq: ["Загальні"]
+  faq: ["app"]
```

Comment:
What is the benefit of using English taxonomy everywhere instead of localized one? Isn't a localized one (present in the URL) improving the SEO by catching relevant search requests?

A translator can add a translation here, just in one place, without referencing something in the config. Isn't it simpler?

Reply:
I think that all FAQ translations should be in a single file for simplicity. If we decide to rename "Map Editing" to "Map Changes" this could be done in a config without touching MD files. Also typos could lead to extra sections. If user writes `"Bookmarks and Tracks"` and `"Bookmark and track"` those would be treated as two different taxonomies.

Disadvantage of such approach: we need to translate all strings even if there is no FAQ pages. For example there is no TTS FAQ page in german but I had to add `faq_section_tts` translation to German language.

And if we add new section such as "ios" it should be translated to all FAQ languages or we'll get Zola errors.

Reply:
The comparison of two approaches:

1. FAQ section/taxonomy translations are stored in config.toml
Pros:
- When translations should be changed, only a single file is edited

Cons:
- URLs do not have translated keywords, they always have English translation key, that is not good for SEO
- When adding a single new FAQ page, all taxonomies in config.toml should be translated if the language is missing, even if these categories are not used/added yet.

2. FAQ section translations are embedded in each FAQ article's taxonomy

Pros:
- Better SEO as the URLs are translated and contain important keywords
- Easier to add new languages/FAQ articles in a new language (no need to translate all FAQ sections, only the currently added one needs translation)
- Simpler to translators and reviewers, they type the translated taxonomy directly into an article, in one file, instead of guessing which translation key from a config should be used (another article in the same section can be copy-pasted and edited)

Cons:
- If a FAQ taxonomy/section should be renamed, all contained pages should be edited (renaming is bad for SEO, not sure that there's an `alias` support here, we may need to add manual aliases in this case)
- Typos in every single file may lead to extra sections (can it be avoided by adding a validator script btw?)


@strump did I miss something? So far the benefits from the second approach